packages/backend/src/managers/catalogManager.ts

```diff
@@ -325,13 +330,13 @@ export class CatalogManager extends Publisher<ApplicationCatalog> implements Dis
       choices.tools = this.filterRecipes(subfilters).choices.tools;
     } else {
       choices.tools = result
-        .map(r => r.backend)
+        .flatMap(r => r.backends)
         .filter(b => b !== undefined)
         .filter((value, index, array) => array.indexOf(value) === index)
         .sort((a, b) => a.localeCompare(b))
```

Comment:
remark (non-blocking): non related to this PR, but this filtering chain is unreadable, what is `r` ? what is `b`?, what is the type of `value`, what is the type of `a` and `b`? :/